tests/test_bingx_api.py

```
@@ -2,159 +2,155 @@
     return BingXAuthenticator(
-        os.getenv('BINGX_KEY', ''),
-        os.getenv('BINGX_SECRET', '')
+        os.getenv("BINGX_KEY", ""),
+        os.getenv("BINGX_SECRET", ""),
     )
 
-def test_public_price():
-    """Test public price endpoint."""
+
+def test_public_price() -> None:
     response = requests.get(
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)